poet/sawtooth_poet/validator_registry/processor/handler.py

```diff
@@ -121,11 +121,9 @@ class ValidatorRegistryTransactionHandler(object):
         '5Jz5Kaiy3kCiHE537uXcQnJuiNJshf2bZZn43CrALMGoCd3zRuo'
 
     def __init__(self):
-        self._report_private_key = \
-            signing.encode_privkey(
-                signing.decode_privkey(self.__REPORT_PRIVATE_KEY_WIF, 'wif'),
-                'hex')
-
+        # Since signing works with WIF-encoded private keys, we don't have to
```

Comment:
Normally, I would agree, but after spending part of my Friday afternoon tracking down why the simulator was failing, and it was because of this canonical key representation change, I would prefer to leave the encode/decode comments in.